compiler/Eta/Utils/Platform.hs

```diff
@@ -28,7 +28,7 @@ data Platform
               -- Word size in bytes (i.e. normally 4 or 8,
               -- for 32bit and 64bit platforms respectively)
               platformWordSize                 :: {-# UNPACK #-} !Int,
-              platformUnregisterised           :: Bool,
+              platformUnregistered           :: Bool,
```

Comment:
Curious about this term. Again, not confident (betting it'll be rejected). If the change is ok, then I'd adjust the indentation.